src/app/static/src/app/components/adr/SelectDataset.vue

```diff
@@ -45,12 +46,12 @@
                     >
                     </label>
                 </tree-select>
-                <div v-if="fetchingDatasets" class="pt-2" id="fetching-datasets">
-                    <loading-spinner  size="xs"></loading-spinner>
+                <div :class="fetchingDatasets ? 'visible' : 'invisible'" style="margin-top:15px;" id="fetching-datasets">
```

Comment:
Modifying the visible/invisible class here rather than using v-if so that the height of the dialog doesn't jump about when finish fetching datasets. Setting the margin style to overcome the loading spinner giving itself a negative top margin. 

The styling of the drop down when disabled is almost identical to when it's enabled - the drop icon is a marginally different shade of grey. So I'm not sure that really helps. But see if you think it's better!

Reply:
I think it's brilliant...

New users will be aware that they are waiting for drop down data.

Reply:
Resolve the conflict and feel free to merge